test/famfs_unit.c

```diff
@@ -97,7 +97,7 @@ int create_mock_famfs_instance(
 	memset(logp, 0, FAMFS_LOG_LEN);
 
 	/* First mkfs should succeed */
-	rc = __famfs_mkfs("/dev/dax0.0", sb, logp, device_size, 0, 0);
+	rc = __famfs_mkfs("/tmp/famfs", sb, logp, device_size, 0, 0);
```

Comment:
why this change? First argument of __famfs_mkfs is the dax device

Reply:
I thought since we created a mock famfs, that should be used to do mkfs.
And also I thouhgt that /dev/dax device may not be available for the unit tests. (that it is available for smokes)

Will change it back...